extensions/csv/src/main/java/io/deephaven/csv/util/CsvReaderException.java

```diff
@@ -0,0 +1,14 @@
+package io.deephaven.csv.util;
+
+public class CsvReaderException extends Exception {
```

Comment:
Food for thought regarding exceptions.  Not asking for you to change anything, just to think about possibilities.
I do believe throwing UncheckedDeephavenException is better than using RuntimeException from this code.  But
an alternative would be to make this class extend RuntimeException thus making it unchecked, and consistently just have and throw that.  In a more complicated world, you could have two of these, one unchecked and one checked, if for some reason we do want to have a checked variant (I am not sure I see an advantage for that, I think I would prefer a world where we only have one Unchecked Exception type that all this code consistently uses in every case).

Reply:
Oh... I made CsvReaderException checked and threaded the throws declarations through everywhere. If you would rather it be unchecked, I can do that too.

Reply:
I don't have a strong position on this, only that I am aware that there are people who push hard against any type of checked exceptions as a usability issue and an interface issue (exposing implementation details in signatures).  See for example: https://softwareengineering.stackexchange.com/questions/177806/decision-for-unchecked-exceptions-in-scala